### PR TITLE
Add version number to footer

### PR DESCRIPTION
**Pull request by jeffdaley**
Adds the Hermes version number and revision to the footer.

CleanShot 2023-08-24 at 19 35 43@2x

Note: This is a follow up from https://github.com/hashicorp-forge/hermes/pull/297

### CODE REVIEW

**Review by anubhavmishra — APPROVED** [review]

File: web/app/routes/application.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import window from "ember-window-mock";
 import { REDIRECT_STORAGE_KEY } from "hermes/services/session";
 import Transition from "@ember/routing/transition";
 import MetricsService from "hermes/services/metrics";
+import { HERMES_GITHUB_REPO_URL } from "hermes/utils/hermes-urls";
```
Comment (anubhavmishra) [inline web/app/routes/application.ts:14]: Not sure if we wanted to use `HERMES_GITHUB_REPO_URL` anywhere?

Comment (jeffdaley) [reply web/app/routes/application.ts:14]: What did you have in mind?

Comment (anubhavmishra) [reply web/app/routes/application.ts:14]: Oh, I wasn't sure if you added it to use it somewhere so I was asking. Otherwise, looks good to me!

Comment (jeffdaley) [reply web/app/routes/application.ts]: Oh whoops, thanks for catching that. I added it while experimenting and forgot to remove.

**Pull request merged**